_data/apis.yml

```diff
@@ -0,0 +1,33 @@
+- openapi: https://raw.githubusercontent.com/teamdigitale/api-openapi-samples/master/openapi-v3/geodati.gov.it.yaml
+  name: Repertorio Nazionale dei Dati Territoriali - geodati.gov.it
+  abstract: Interfaccia REST verso il repertorio nazionale dei dati territoriali.
+  contact:
+    name: AgID
+    logo: https://i.imgur.com/sNoUqR4.png
+  tags: []
+  language: [it]
+- openapi: https://raw.githubusercontent.com/teamdigitale/api-openapi-samples/master/openapi-v3/api.daf.teamdigitale.it.yaml
+  name: Data & Analytics Framework CKAN API
+  abstract: API per ricercare e visualizzare gli open data del Data & Analytics Framework in api.daf.teamdigitale.it
+  contact:
+    name: Team Digitale
+    logo: https://i.imgur.com/OTNbMVs.png
```

Comment:
dobbiamo usare per forza imgur? credi sia possibile ospitare le immagini in locale?